docs/pages/debugging/devtools-plugins.mdx

```diff
@@ -176,6 +176,36 @@ export default function App() {
 
 In the terminal, run `npx expo start`, press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of dev tools, and then select the React Query plugin. This will open the plugin's web interface, displaying queries as they are used in your app.
 
+### Redux DevTools
+
+Based on the full [Redux DevTools](https://github.com/reduxjs/redux-devtools/) (from the Chrome extension) but as an Expo Dev Plugin.
+
+- Easy to install and run.
+- Live list of actions and how it affects the state.
+- Ability to rewind, replay, and dispatch actions from the devtools.
```

Comment:
```suggestion
The `redux-devtools-expo-dev-plugin` is based on the [Redux DevTools](https://github.com/reduxjs/redux-devtools/) (from the Chrome extension). It provides a live list of actions and how they affect the state, and the ability to rewind, replay, and dispatch actions from the DevTools.
```

I updated the this sub-section because I don't think list item no. 1 is worth mentioning and we can introduce the plugin in the first line as the main subject matter here. Then, we can call the features this plugin has within the same paragraph.